hibike/hibikeDevices.json

```diff
@@ -67,6 +61,26 @@
                                                      ]
     },
 
+
+    {"id": 10,     "name": "YogiBear",      "params": [
```

Comment:
@yizow YogiBear PR is now merged. You should be able to rebase this branch to get the json changes, instead of making them yourself.